### ISSUE TITLE
[HDRP] Setting material to Transparent gives shader error

### ISSUE THREAD

**Issue by ThomasAunvik** [Unity-Technologies/ScriptableRenderPipeline#1406]
# Versions

Unity Version: 2018.1.1f1
HDRenderPipeline Version: 1.1.8-preview
SRP Core Version: 1.1.8-preview

i also have Lightweight RP Installed: 1.1.8-preview. HDRP is set in the Graphics settings.

# Description

When creating a new material (HDRenderPipeline/Lit) and selecting Surface Type to Transparent, then every Lit material turns to pink and getting this error.

```
Shader error in 'HDRenderPipeline/Lit': failed to open source file: 'CoreRP/ShaderLibrary/Common.hlsl' at line 282 (on d3d11)

Compiling Vertex program with UNITY_PASS_SHADOWCASTER _NORMALMAP_TANGENT_SPACE _SURFACE_TYPE_TRANSPARENT _BLENDMODE_ALPHA _BLENDMODE_PRESERVE_SPECULAR_LIGHTING _ENABLE_FOG_ON_TRANSPARENT
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_FULL_HDR
```

I have managed to reproduce it 100%.

Inspector also changes:
![image](https://user-images.githubusercontent.com/9968151/40551936-95aff122-603e-11e8-835d-b472e0d2d144.png)

It fixes itself by relaunching unity, every other lit object in the project view shows working except the transparent object. When selecting the material in the project view, the same issue happens. This is when i have not set any material to it.

Next is when i set this material to any object and relaunching unity, the same thing happens, including the error.

**Comment by sebastienlagarde** [comment]
Thanks for reporting. This is a know issue and a backport of the fix is in progress for 2018.1. Workaround is to re-import / restart

**Comment by ThomasAunvik** [comment]
Alright
